carbonmark/components/Transaction/index.tsx

```diff
@@ -75,19 +80,12 @@ export const Transaction: FC<Props> = (props) => {
           amount={props.amount}
           price={props.price}
           description={props.submitText}
-          spenderAddress={getAddress("carbonmark")}
+          spenderAddress={props.spenderAddress}
           onSubmit={props.onSubmit}
           onClose={props.onCancel}
           status={props.status}
         />
       )}
-      {!!props.onGoBack && (
```

Comment:
This Go Back Button was never included in any designs.
Still, wenn a transaction fails and the error message appears, the user should be able to go back to the former modal (not the transaction modal) to change the input values again.
This is the case for Editing a Listing for example whose inputs are also appearing in a modal (modal after modal). 

Just to be sure that this is still working.

Reply:
This was the design I had with the 'Go Back' Button:
![Screen Shot 2023-05-19 at 10 19 32 AM](https://github.com/KlimaDAO/klimadao/assets/72105234/6564bb67-a068-4263-ac19-8eb32c168d87)

I had moved it up to the Retire Modal but I'll move it back to Transaction